### PR TITLE
Restructure site

### PR DESCRIPTION
**Pull request by pwright**
opened to correct some issues in https://github.com/aerogear/datasync/pull/222

### CODE REVIEW

File: local-antora-playbook.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+site:
```
Comment (wtrocki) [inline local-antora-playbook.yml]: +100 - I was missing that!

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ The Voyager framework has a small ecosystem of components for building GraphQL b
 
 ## Local Development Setup
 
-The [Local Development Guide](./doc/guides/local-development.md) will help contributors to get started developing Voyager Server.
+The [Local Development Guide](./docs/guides/local-development.md) will help contributors to get started developing Voyager Server.
```
Comment (pwright) [inline README.md]: @wtrocki  looks like this link is invalid too? should i remove mention of local dev guide?

File: docs/integreatly/modules/ROOT/pages/js-client-offline.adoc [hunk 3]
```diff
@@ -265,13 +263,13 @@ client.mutate<Task>(options);
 
 === Using cache update helpers for subscriptions
 
-The @aerogear/voyager-server module provides a subscription helper which can generate the necessary options to be used with Apollo Client's `subscribeToMore` function.
+The @aerogear/voyager-client module provides a subscription helper which can generate the necessary options to be used with Apollo Client's `subscribeToMore` function.
```
Comment (pwright) [inline docs/integreatly/modules/ROOT/pages/js-client-offline.adoc]: lol, I'm wondering how did this change end up in this PR

File: README.md (hunk 2 above)
Comment (wtrocki) [reply README.md]: I have made this change. I should use resources

File: docs/integreatly/modules/ROOT/pages/js-client-offline.adoc (hunk 3 above)
Comment (wtrocki) [reply docs/integreatly/modules/ROOT/pages/js-client-offline.adoc]: There is some strange git stuff happening here.

File: README.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ The Voyager framework has a small ecosystem of components for building GraphQL b
...
```
Comment (wtrocki) [inline README.md]:
```suggestion
The [Local Development Guide](./resources/local-development.md) will help contributors to get started developing Voyager Server.
```

**Pull request merged**